sql/core/src/test/resources/sql-tests/results/csv-functions.sql.out

```diff
@@ -24,7 +24,7 @@ select from_csv('1', 1)
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-Schema should be specified in DDL format as a string literal or output of the schema_of_csv function instead of 1;; line 1 pos 7
+The expression '1' must be evaluated to a valid string.;; line 1 pos 7
```

Comment:
`valid schema string`?

Reply:
and as an error message, seems better to have `The expression '1' is not a valid schema string.`

Reply:
replaced